projects/canopy/src/lib/repsonsive/repsonsive.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
I'm not too sure about adding the responsive module, I would just keep them separate 🤔  
The two directives are useful but I don't see them being used too often in a codebase and most of the time you will use either one or the other in a module/component.

Reply:
I was a bit on the fence about this too but leaning a bit toward having them available as one module, partly because it's consistent with the `SpacingModule` and partly because it would be convenient to choose to have them imported together or separate. Having said that, I do kinda agree with your points too 🙂 

@andy-polhill  - what do you think?

Reply:
Mmm I'm actually more keen to also remove the spacing module :D i personally only always add each module separately 🤔

Reply:
I think I'm inclined to trend in the same direction as @elenagarrone on this one. I think the aggregating modules are maybe a bit of an anti-pattern.

Reply:
I can see the logic in that :) I'll remove the responsive module and keep them separate 👍

Reply:
This is done now ✅